includes/Service.php

```diff
@@ -38,4 +46,33 @@ public function disable() {
 	public function is_enabled() {
 		return 'true' === get_option( 'nfd_coming_soon', 'false' );
 	}
+
+	/**
+	 * Create/update the last changed timestamp.
+	 *
+	 * @return void
+	 */
+	public function last_changed_timestamp() {
```

Comment:
I think we should make this private, as nobody should be able to set this externally.

Reply:
Update this.